api/docs/developer-system-requirements.md

```diff
@@ -2,7 +2,7 @@
 ### Operating System
 We currently only support MacOS and Linux for development and testing.
 ### Prerequisite Software
-  * [Java 8](https://openjdk.java.net/projects/jdk8/)
+  * [Java 8](https://adoptium.net/temurin/releases/)
```

Comment:
"Our July 2021 and future releases will come from [Adoptium.net](https://www.adoptium.net/)"
https://adoptopenjdk.net/